lib/components/Input/TextArea.module.scss

```diff
@@ -13,6 +13,9 @@ $line-height: 5*$grid-size;
         left: 0px;
         right: 0px;
         visibility: hidden;
+        white-space: pre-wrap;
+        overflow-wrap: break-word;
+        padding-right: 6*$grid-size; // accounts for scroll bar

```

Comment:
Does this work with RTL?

Reply:
Yes, this is just for sizing since the ghost text area is hidden